fmriprep/cli/run.py

```diff
@@ -173,13 +173,18 @@ def get_parser():
                        help='EXPERIMENTAL/TEMPORARY: Use SyN correction in addition to '
                        'fieldmap correction, if available')
 
+    # Surface generation
+    g_surfs = parser.add_mutually_exclusive_group()
```

Comment:
I will say this is a little dissatisfying. Is it possible to make subgroups, as in `g_surfs = g_fs.add_mutually_exclusive_group`?

<img width="733" alt="screen shot 2018-04-06 at 10 25 38" src="https://user-images.githubusercontent.com/83442/38426552-f572a5ae-3984-11e8-9790-b6b5bc974015.png">

Reply:
Just checked:

```Python
In [5]: parser = argparse.ArgumentParser()

In [6]: g_surf = parser.add_argument_group('Surface-preprocessing options')

In [7]: g_surf_xor = g_surf.add_mutually_exclusive_group()

In [8]: g_surf.add_argument('--no-submm-recon', action='store_false', dest='hires',
   ...:                        help='disable sub-millimeter (hires) reconstruction')
   ...:                        
Out[8]: _StoreFalseAction(option_strings=['--no-submm-recon'], dest='hires', nargs=0, const=False, default=True, type=None, choices=None, help='disable sub-millimeter (hires) reconstruction', metavar=None)

In [9]: g_surf_xor.add_argument('--fs-no-reconall', '--no-freesurfer',
   ...:                          action='store_false', dest='run_reconall',
   ...:                          help='disable FreeSurfer surface preprocessing.
   ...: '
   ...:                          ' Note : `--no-freesurfer` is deprecated and will be removed in 1.2.'
   ...:                          ' Use `--fs-no-reconall` instead.')
   ...:                          
Out[9]: _StoreFalseAction(option_strings=['--fs-no-reconall', '--no-freesurfer'], dest='run_reconall', nargs=0, const=False, default=True, type=None, choices=None, help='disable FreeSurfer surface preprocessing. Note : `--no-freesurfer` is deprecated and will be removed in 1.2. Use `--fs-no-reconall` instead.', metavar=None)

In [10]: g_surf_xor.add_argument('--cifti-output', action='store_true', default=
    ...: False,
    ...:                          help='output BOLD files as CIFTI dtseries')
    ...:                          
Out[10]: _StoreTrueAction(option_strings=['--cifti-output'], dest='cifti_output', nargs=0, const=True, default=False, type=None, choices=None, help='output BOLD files as CIFTI dtseries', metavar=None)

In [12]: print(parser.format_help())
usage: ipython [-h] [--no-submm-recon] [--fs-no-reconall | --cifti-output]

optional arguments:
  -h, --help            show this help message and exit

Surface-preprocessing options:
  --no-submm-recon      disable sub-millimeter (hires) reconstruction
  --fs-no-reconall, --no-freesurfer
                        disable FreeSurfer surface preprocessing. Note :
                        `--no-freesurfer` is deprecated and will be removed in
                        1.2. Use `--fs-no-reconall` instead.
  --cifti-output        output BOLD files as CIFTI dtseries
```